javascript/README.md

```diff
@@ -6,6 +6,10 @@
 
 Each type of code challenge has slightly different instructions. Please refer to the notes and examples below for instructions for each DS&A assignment type.
 
+## Table of Contents
+
+![Whiteboard](./array-reverse/array-reverse.png)
```

Comment:
move your photo to your `array-reverse/README.md` and replace this line with `*  [Array Reverse](javascript/array-reverse/README.md)`  which look something like this: 
*  [Array Reverse](https://github.com/vbchomp/data-structures-and-algorithms/blob/array-reverse/javascript/array-reverse/README.md)